sled-agent/src/rack_setup/service.rs

```diff
@@ -480,6 +481,25 @@ impl ServiceInner {
                 for svc in &zone.services {
                     let kind = match svc {
                         ServiceType::Nexus { external_ip, internal_ip: _ } => {
+                            // NOTE: Eventually, this IP pool will be entirely
+                            // user-supplied. For now, however, it's inferred
+                            // based on the input IP addresses.
```

Comment:
Just checking -- `external_services_ip_pool` is a `Vec` with exactly one element now, because it's just made from the IP address we provide for Nexus. But it's a `Vec` so that the operator can provide multiple ranges in future, yes?